corgi/core/constants.py

```diff
@@ -49,11 +49,15 @@
     "ProductVariant": "product_variants",
 }
 
-# Filter on "root components": SRPMs, index container images, or Github repos for managed services
+# Filter on "root components":
+# SRPMs, modules, index container images, or Github repos for managed services
 SRPM_CONDITION = Q(type="RPM", arch="src")
+RPMMOD_CONDITION = Q(type="RPMMOD")
```

Comment:
should we not also include arch="noarch" ?

Reply:
I can see that logic (and prev status quo)  - though today we have a closed set of constraints (OCI-noarch, RPM-src, etc) on other root components and things can always change in the future ... when I see 2 equal choices I go with the more constrained choice as it is safer (and more contained).